### PR TITLE
Lisp Ubuntu 20.04

### LINKED ISSUE #256: Update lisp image to support new Ubuntu version

### PR DESCRIPTION
**Pull request by ArDrambyan**
Closes #256 

## Changes

Updated base image for Lisp to be derived from Ubuntu 20.04

### CODE REVIEW

**Comment by ArDrambyan** [comment]
Coderunners https://rundash-staging.codesignal.com/coderunner-test/142.93.115.222/test-results and https://rundash-staging.codesignal.com/coderunner-test/159.203.191.230/test-results are passing all the tests

File: lisp/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,9 @@
-FROM codesignal/ubuntu-base:v8.0.1
+FROM codesignal/ubuntu-base:new
 
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
+  libncurses5 \
```
Comment (menewman) [inline lisp/Dockerfile:5]: You know I have to ask "why" on these lib additions 😄 

Is there any documentation that you referenced for this installation that we could reference? Will help give guidance on future changes, if so

Comment (ArDrambyan) [reply lisp/Dockerfile:5]: During apt-get I just got missing dependencies error for the Clisp and I just added those dependencies into the installation rule 😅

Comment (menewman) [reply lisp/Dockerfile:5]: Fair enough

**Review by menewman — APPROVED** [review]

**Pull request merged**